### DISCUSSION TITLE
Troubles with tp.file.cursor

### DISCUSSION THREAD

**Discussion by ExceedingLife** [SilentVoid13/Templater#628] [Help]
Hello all,

I have made a few templates and I would love it if I could specify the cursor position after I insert the template into the note. I am having troubles on figuring out how to do this.
Here is an example of my template and then this function where it should set the cursor position.

I enabled the sliders in templater settings.

Anything I am doing wrong I would love to know how to do it correctly.

```
<div class="fs12"><b><% tp.file.cursor() %></b></div>
```

Ive tried it with (0)  (1) ...

I also tried it with other templates but have never gotten it to move to this position.
When I insert it this file.cursor stays on the note and doesn't disappear.

**Comment by AB1908** [comment]
Cursor stuff does seem to be having issues. There are a few open issues for it IIRC.

**Reply by ExceedingLife** [reply]
any idea when a fix will happen for this? im excited to be able to use this cursor in templater itll make my note taking alot easier

**Reply by AB1908** [reply]
Can't really say here. Liam and shabegom seem to be the only ones contributing ATM and they're fairly swamped.

**Comment by welpdx** [comment]
Is it working now? I've tried this and it seems to be working for me...

> I enabled the sliders in templater settings.

That is the `automatic jump to cursor` slider right?
